python/cudf/dataframe/multiindex.py

```diff
@@ -42,8 +43,10 @@ def __init__(self, levels, codes=None, labels=None, names=None):
             column_names = list(range(len(codes)))
```

Comment:
@quasiben You had changed this to `list(range(len(levels)))`. Had to revert back to using `codes` because of failing tests where length of codes was more than length of levels and we try to index column_names out of bounds. https://github.com/rapidsai/cudf/pull/1659/files#diff-00d8f38fb2f9fcb4bca68211490bc126R59.

See failing test example:
 https://github.com/rapidsai/cudf/blob/branch-0.8/python/cudf/tests/test_multiindex.py#L26

Need to discuss further if using `levels` is the way to go.

Reply:
@quasiben Decision is to continue using `codes` instead of `levels`